pkg/analyser/analyzer.go

```diff
@@ -34,10 +40,22 @@ func (a Analyzer) Analyze(remoteResources []resource.Resource, resourcesFromStat
 				Res:       stateRes,
 				Changelog: delta,
 			})
+			for _, d := range delta {
```

Comment:
Maybe we could move this in a separate unexported method to avoid a too complex `Analyze` method

Reply:
I agree 👌🏻

Reply:
Done 7f5211c